manala.apt/vars/main.yml

```diff
@@ -163,6 +163,9 @@ manala_apt_repositories_patterns:
   mariadb_10_2:
     source: deb http://ftp.osuosl.org/pub/mariadb/repo/10.2/debian {{ ansible_distribution_release }} main
     key: "{{ (ansible_distribution_release not in ['wheezy', 'jessie'])|ternary('mariadb','mariadb_legacy') }}"
+  mariadb_10_3:
+    source: deb http://ftp.osuosl.org/pub/mariadb/repo/10.3/debian {{ ansible_distribution_release }} main
+    key: "{{ (ansible_distribution_release not in ['wheezy', 'jessie'])|ternary('mariadb','mariadb_legacy') }}"
```

Comment:
I'm not sure this tweak is still relevant with the new repo. Could you have a look ?